examples/chip-tool/commands/tests/Commands.h

```diff
@@ -70,8 +67,8 @@ class TestCluster : public TestCommand
     }
 
 private:
-    std::atomic_uint16_t mTestIndex;
```

Comment:
Unfortunately, this is going to cause more CI failures on Darwin again until the Darwin situation is actually fixed.  :(

Reply:
This is ironic, since I've been doing all my testing on Darwin, but the change to shift Darwin away from POSIX to libdispatch just came yesterday.

I'm going to revert this revert so that you end up where you were yesterday, and this PR won't make it worse (hopefully).